### ISSUE TITLE
Introduce a long `--config` alias for `-C`

### ISSUE THREAD

**Issue by cratelyn** [fastly/Viceroy#3]
Currently, we provide a `-C` option for providing the path to configuration for Viceroy (_see below_). We should add a longer `--config` option for those who prefer that style.

https://github.com/fastly/Viceroy/blob/main/cli/src/opts.rs#L33

Reported by @peterbourgon. :slightly_smiling_face:

**Comment by peterbourgon** [comment]
Nice, thank you!! I guess just curiosity, but why `-C` and not `-c`?

**Comment by cratelyn** [comment]
No _strong_ reasoning there! We used `-C` rather than `-c` because ime that seemed more common for specifying a path to a config of some sort, i.e. `git`. I recall `-c` being a flag used for things like color, or counting, when I took a look around, and the hope was to keep the `-[a-z]` space as open as possible.

Most likely, `-C` (or `--config`!) will become invisible once we've integrated more closely with the Fastly CLI.
